src/test/java/com/imagination/cbs/service/impl/ReasonsForRecruitingServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package com.imagination.cbs.service.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import com.imagination.cbs.domain.ReasonsForRecruiting;
+import com.imagination.cbs.dto.ReasonsForRecruitingDto;
+import com.imagination.cbs.mapper.ReasonsForRecruitingMapper;
+import com.imagination.cbs.repository.ReasonsForRecruitingRepository;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReasonsForRecruitingServiceImplTest {
```

Comment:
apart from asserting expected outputs .. we can have some more tests to verify methods calls .. like called atleast once etc.